### DISCUSSION TITLE
Feature Request: Add bootstrap styling

### DISCUSSION THREAD

**Discussion by Restingo** [wire-elements/spotlight#55] [Ideas]
Add bootstrap theme for styling.

**Comment by PhiloNL** [comment]
Hi @Restingo,
Have you tried setting `include_css` to `true`? This will include the required CSS for Spotlight and should work with other CSS frameworks like Bootstrap.

**Reply by nam-co** [reply]
Hi Philo, I tried include_css but it changes all the bootstrap design, maybe an option is to add a basic css/scss file with no "body, html, etc... styles"? just the what's needed for the box and results

**Comment by Restingo** [comment]
Hi @PhiloNL, thanks for your fast reply! I've tried it but it destroys my template (I'm using AdminLTE from https://github.com/jeroennoten/Laravel-AdminLTE) because it includes Normalize.css I think?

**Comment by Restingo** [comment]
![image](https://user-images.githubusercontent.com/84022569/124028902-d1cc3200-d9f4-11eb-9845-180e7dd165a9.png)

Left side is without css, right side is with css.

**Comment by Restingo** [comment]
@PhiloNL any idea?

**Comment by TechTailor** [comment]
Would also appreciate a way to enable/add custom css styling.

**Comment by modernben** [comment]
This blows up styling for my site as well. I had to add the `@livewire('livewire-ui-spotlight')` in my head before I start loading styles. Perhaps a way to publish the CSS and then have a blade directive for that CSS specifically instead of the component pulling it in? Something like `@spotlightStyles`?

If there's a general consensus, I can get started on a PR.

**Comment by eafarooqi** [comment]
Still no support for bootstrap? setting include_css to true destroy the bootstrap template. Need to have custom css which should not be tailwindcss or full support for bootstrap.
